app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -1486,10 +1495,10 @@ class StatePlayerRecyclerViewAssembler private constructor(
     private val delayShowAdditionalHintsMs: Long,
     private val delayShowAdditionalHintsFromWrongAnswerMs: Long
   ) {
-    var trackedWrongAnswerCount = 0
+    var trackedWrongAnswerCount = DEFAULT_WRONG_ANSWER_COUNT
     var previousHelpIndex: HelpIndex = HelpIndex.getDefaultInstance()
```

Comment:
It's worth also having a default for HelpIndex. Note that this should be fine to save as a const val since proto lite objects are effectively immutable.

Reply:
I tried this earlier but there is an error for that.
![image](https://user-images.githubusercontent.com/54740946/107842991-570bcd00-6ded-11eb-8e1b-934f0694f916.png)

Reply:
Ah, I was wondering about that (couldn't find any quick results when I searched before leaving my comment). Could we instead make it a constant within the assembler class rather than putting it outside? We don't need to mark it as const--a val proto message is guaranteed to be constant.

Reply:
Done